### ISSUE TITLE
HistGradientBoostingClassifier - something to try for Q3 and Q4

### ISSUE THREAD

**Issue by danieltomasz** [phuycke/ML-in-Python#7]
I will push the results with MVE but I think we need to improve it. I will be able to work on it, but only starting from Thursday afternoon.

https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html
https://machinelearningmastery.com/gradient-boosting-with-scikit-learn-xgboost-lightgbm-and-catboost/
